src/screens/PutawayList/index.tsx

```diff
@@ -144,13 +144,23 @@ class PutawayList extends React.Component<Props, State> {
     });
   };
   onPutAwayTapped = (putAway: PutAway) => {
-    this.props.navigation.navigate('PutawayDetail', {
-      // putAway,
-      putAway: putAway,
-      exit: () => {
-        this.props.navigation.navigate('PutawayList');
-      },
-    });
+    console.log('putAway ::', putAway);
+    if (putAway?.putawayItems?.length > 1) {
+      this.props.navigation.navigate('PutawayDetail', {
+        putAway: putAway,
+        exit: () => {
```

Comment:
Do it as: `exit: () => this.props.navigation.navigate('PutawayList')`

Reply:
@awalkowiak it's an IDE auto formation. it's automatically added into the next line and I don't think is there has any issue with line formations.

Reply:
![image](https://user-images.githubusercontent.com/35322132/144608074-ad3f0548-ac81-4f06-b549-195875c51686.png)
@awalkowiak

Reply:
@hellochirag in my request there is no curly brackets

Reply:
but fine, lets go with it